v2/pkg/stratus/runner/runner.go

```diff
@@ -84,6 +84,8 @@ func (m *Runner) WarmUp() (map[string]string, error) {
 	log.Println("Warming up " + m.Technique.ID)
 	outputs, err := m.TerraformManager.TerraformInitAndApply(m.TerraformDir)
 	if err != nil {
+		log.Println("Error during warm up. Cleaning up technique prerequisites with terraform destroy")
+		_ = m.TerraformManager.TerraformDestroy(m.TerraformDir)
```

Comment:
might make sense to `defer` it here? no strong opinions

Reply:
Interesting thought; could do something like the below (keeping the `_ =` to avoid 'unhandled error' warnings).
However, I believe it won't make a difference because there's is nothing in the way that could fail and thus prevent the `.TerraformDestroy()` from being called. Happy to stand corrected though ;)

```go
		defer func() {
			_ = m.TerraformManager.TerraformDestroy(m.TerraformDir)
		}()
```

Reply:
I was thinking of simply `defer m.TerraformManager.TerraformDestroy(m.TerraformDir)`, but it might trigger an IDE warning instead. No strong opinion, let's keep it as is